packages/backend/src/index.ts

```diff
@@ -768,6 +769,11 @@ export const start = async (
 
   Model.knex(knex)
 
+  // Update Operator Tenant from config
+  const tenantService = await container.use('tenantService')
+  const error = await tenantService.updateOperatorApiSecretFromConfig()
+  if (error) throw error
+
```

Comment:
This is happening in the `start` method after we run migrations and start the admin server, etc.

Is this the correct behavior? It will error if the operator id from the env var is not found (tenant service returns an error), or some unknown error throws. The not found error could be from a bad environment variable input.

This will show an error log and exit the process. I debated catching and logging at warning level because strictly speaking the app will still work. Operator tenant id isnt (at this point) used for anything else in the app (just used in the initial migration and here). So it can work without this but it could be in an unexpected state (still with the old api secret). You wouldnt know unless dutifully watching the logs.

Reply:
Thinking about it more, we throw an error when parsing the config if `OPERATOR_TENANT_ID` is not set. So I think its probably correct to throw here if we know its bad.